_plugins/hooks/config.rb

```diff
@@ -1,8 +1,8 @@
 Jekyll::Hooks.register :site, :after_init do |site|
-  if ENV['NETLIFY']
+  if ENV['NETLIFY'] && ENV['CRDS_ENV'] === 'prod'
```

Comment:
NETLIFY is a "netlify read only variable" that come free from netlify basically is a boolean value to answer is this a build coming hosted from netlify